operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -246,6 +246,19 @@ Power on and start management services on the HPE Cray EX management Kubernetes
     session running in detached mode. The `sat bootsys` command will automatically exit screen
     sessions when nodes have finished booting.
 
+1. (`ncn-m001#`) Confirm all NCNs have booted.
```

Comment:
This step seems unnecessary as the previous `sat bootsys` command will ensure the nodes are reachable by SSH before exiting. More generally, it would be nice to reduce the double-checking of things in the documentation unless there are known issues in the automation, and in that case, we should fix the automation where we can.

Reply:
This is here because I have observed sysadmins wanting a simple way to confirm power up before moving forward.  It validates that ssh is functional from ncn-m001 to the other management nodes.  If that SSH is not functional, there will be other symptoms seen later that would root cause to this point and the failure which could have first been discovered here.

Not changing this section.